app/services/serializer_version_chooser.rb

```diff
@@ -0,0 +1,17 @@
+# frozen_string_literal: true
+
+class SerializerVersionChooser
+  DEFAULT_API_VERSION = 'V2'
+
+  def self.call(interpolatable)
+    serializer = "V2::#{interpolatable}Serializer"
+
+    serializer = if const_defined?(serializer)
+                   serializer
+                 else
+                   serializer.sub('V2::', '')
+                 end
+
+    serializer.constantize
```

Comment:
Could you add coverage for an undefined V1 serializer here too? I'm curious what the behaviour would be - whatever it is it's probably useful to document via a spec. e.g. `SerializerVersionChooser.call(:foo)`

Reply:
Never mind, I see you already have this covered 👍